src/app/navigation/Home.screen.tsx

```diff
@@ -8,30 +15,35 @@ export type HomeRouteParams = undefined;
 
 export const Home = () => {
   const [movies, setMovies] = useState([]);
+  const [nbMovies, setNbMovies] = useState<number>();
 
   useEffect(() => {
-    const fetchMovies = async () => {
-      try {
-        const response = await fetch(API_URL);
-        const json = await response.json();
-        setMovies(json.results);
-      } catch (error) {
-        console.error(error);
-      }
-    };
-
-    fetchMovies();
+    fetch(API_URL)
```

Comment:
what if we want to implement pull to refresh or else. we should extract the fetching logic into a method we handle also the loading state